simcon_project/simcon_project/settings.py

```diff
@@ -143,8 +144,19 @@
 LOGOUT_REDIRECT_URL = 'Login'
 LOGIN_URL = 'Login'
 
-# Bootstrap Template for Django Tables 
+# Bootstrap Template for Django Tables
 DJANGO_TABLES2_TEMPLATE = "django_tables2/semantic.html"
 MEDIAFILES_DIRS = [
     os.path.join(BASE_DIR, 'static/media')
 ]
+
+
+#we will need to hide this in the final version
+SENDGRID_API_KEY = os.getenv('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')
+EMAIL_HOST = 'smtp.sendgrid.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'apikey'
+EMAIL_HOST_PASSWORD = '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'
```

Comment:
If this gets merged in, make sure to create or add to the ticket with all the places this is hardcoded

Reply:
made a ticket for this. so far I think this is the only spot.